panacea/config/config.exs

```diff
@@ -19,7 +19,7 @@ config :logger, :console,
   metadata: [:request_id]
 
 config :panacea, :asclepius,
-  uri: URI.parse("http://asclepius:5000"),
+  uri: URI.parse("http://localhost:5000"),
```

Comment:
But when developing locally, you want to be doing `mix phoenix.server` and not having to rebuild images and run compose every time. The way I was doing things with it is `docker run asclepius` and mapping the ports onto localhost and then getting benefits of having the code reloader going